arithmetization/src/main/java/net/consensys/linea/sequencer/txselection/LineaTransactionSelectorConfiguration.java

```diff
@@ -15,128 +15,22 @@
 
 package net.consensys.linea.sequencer.txselection;
 
+import lombok.Builder;
+import lombok.Getter;
+
 /** The Linea configuration. */
+@Getter
+@Builder
 public final class LineaTransactionSelectorConfiguration {
```

Comment:
Should this `class` be a `record`?

Reply:
yes it can be, just found that Lombok builder works with records too, so we can have the benefit of the "named parameters constructor" offered by builders

Reply:
done